tools/distpartitioning/convert_partition.py

```diff
@@ -18,6 +18,36 @@ def create_dgl_object(graph_name, num_parts, \
     This function creates dgl objects for a given graph partition, as in function
     arguments. 
 
+    The input schema object is structure as follows: 
+    1. "nid" is a dictionary, which has the following structure. 
+        "ntype-name" : {
+            "format" : "csv", 
+            "data" : [
+                    [ <path-to-file>/ntype0-name-0, start_id, end_id], 
+                    [ <path-to-file>/ntype0-name-1 start_id, end_id)
```

Comment:
can you explain what are "ntype0-name-0" and "ntype0-name-1"?

Reply:
I'm confused. i checked the doc and it says each of the files defines the metadata file for a partition. why are there only two files? if there should be p files, i think you should illustrate it with p partitions.

Reply:
however, your code below suggests that there are two files for each node type.

Reply:
Done.